src/components/common/Navbar/index.tsx

```diff
@@ -36,11 +36,7 @@ const NavScrollExample = () => {
               </NavLink>
             )}
             {me && (
-              <NavLink
-                className="nav-link"
-                to="/login"
-                onClick={() => me && logOut(me)}
-              >
+              <NavLink className="nav-link" to="/login" onClick={logOut}>
```

Comment:
esto no me convence... tiene un to a login pero hace un logout?
Creo que no debería hacer el cambio de pagina hasta finalizar con la petición asincrónica.